distdir_deps.bzl

```diff
@@ -351,45 +351,49 @@ DIST_DEPS = {
             "remotejdk11_linux_for_testing",
             "openjdk11_linux_archive",
         ],
-        "archive": "zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz",
-        "sha256": "b8e8a63b79bc312aa90f3558edbea59e71495ef1a9c340e38900dd28a1c579f3",
-        "strip_prefix": "zulu11.50.19-ca-jdk11.0.12-linux_x64",
+        "archive": "zulu11.52.13-ca-jdk11.0.13-linux_x64.tar.gz",
```

Comment:
Some of these set `"archive"`, and some don't. Is this part actually needed?

Reply:
If the dep is used in distdir_tar rule, then archive needs to be set.